### PR TITLE
Update the DOE commercial prototype building models test files

### PR DESCRIPTION
**Pull request by lymereJ**
Pull request overview
---------------------
Update the DOE commercial prototype building models test files (based on 90.1-2019 instead of 90.1-2016).

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@lymereJ @lgentile it has been 28 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
This looks like a clean set of changes to the test files, not even any diffs.  The CI results would be all clean if develop was pulled in.  I am partially inclined to not waste those cycles, but I'd like to leave the door open for @JasonGlazer or @mjwitte or anyone else to take a deeper dive into the file changes if interested, so I'll go ahead and get fresh results anyway.

**Comment by Myoldmopar** [comment]
(There are some Github Actions failures over in the fork where this branch actually lives that aren't showing up on the PR.  I think the errors are related to the repo name changing, but I could be wrong.  Anyway, they aren't significant here.)

**Comment by Myoldmopar** [comment]
@lymereJ 2 of the new/modified input files are timing out in our debug build.  This most likely indicates there is something wrong in the file (too many max hvac iterations or something).  We need to figure out how to either simplify some aspect of the file or clean up runtime issues if there are any.  We cannot have our test files taking multiple hours to run for every pushed commit of changes.  Let me know if you want to discuss.

**Comment by lymereJ** [comment]
@Myoldmopar - Really sorry about that, we did run them locally and most completed under 30 min (mostly between 0-10 min). Are you able to point out which one is timing out? I suspect that it is the high rise apartment model as it took 53 min to run on my machine.

**Comment by nrel-bot-3** [comment]
@lymereJ @lgentile it has been 28 days since this pull request was last updated.

**Comment by nrel-bot-2b** [comment]
@lymereJ @lgentile it has been 28 days since this pull request was last updated.

**Comment by mitchute** [comment]
What's the status here? Are you still targeting this release? Did the timing issues get resolved?

**Comment by lymereJ** [comment]
@mitchute - We are waiting for review. Yes, this release would be great. I think that the last CI results show that updated files that were timing out aren't anymore.

**Comment by mitchute** [comment]
@lymereJ that's fine. Can you resolve these conflicts so we can get a fresh CI run? Let me know if you need help with that.

**Comment by rraustad** [comment]
So what was changed here? Wall R, equip efficiency, etc. I can't see anything on the Git diffs. I get the 2016 -> 2019 updates but can we get a summary of changes, a description, etc. I don't see a related issue so there is nothing to reference, except 90.1 and I'd still like to see a summary for these changes as documentation.

File: testfiles/CMakeLists.txt [hunk 1]
```diff
@@ -688,23 +688,23 @@ add_simulation_test(IDF_FILE WCE_Interior_VB_-45_deg.idf EPW_FILE USA_IL_Chicago
...
-add_simulation_test(IDF_FILE ASHRAE9012016_OfficeSmall_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_OutPatientHealthCare_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RestaurantFastFood_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RestaurantSitDown_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RetailStandalone_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_RetailStripmall_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_SchoolPrimary_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_SchoolSecondary_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
-add_simulation_test(IDF_FILE ASHRAE9012016_Warehouse_Denver.idf EPW_FILE USA_CO_Golden-NREL.724666_TMY3.epw)
+# ASHRAE 90.1-2019 DOE commercial prototype models
```
Comment (rraustad) [inline testfiles/CMakeLists.txt:698]: So we are deleting the 90.1 2016 prototypes from this repo. That's OK, but are these files saved somewhere for future use? A DOE web site?

**Comment by jiangithub** [comment]
> So what was changed here? Wall R, equip efficiency, etc. I can't see anything on the Git diffs. I get the 2016 -> 2019 updates but can we get a summary of changes, a description, etc. I don't see a related issue so there is nothing to reference, except 90.1 and I'd still like to see a summary for these changes as documentation.

The technical support document [Energy Savings Analysis: ANSI/ASHRAE/IES Standard 90.1-2019](https://www.energycodes.gov/sites/default/files/2021-07/Standard_90.1-2019_Final_Determination_TSD.pdf) (PDF page 49, Appendix B: Modeling of Individual Addenda) provides a summary of changes from the 90.1-2016 prototypes to 90.1-2019.

File: testfiles/CMakeLists.txt (hunk 1 above)
Comment (jiangithub) [reply testfiles/CMakeLists.txt:698]: All versions of the 90.1 prototype models are saved at the DOE's Building Energy Codes Program [website](https://www.energycodes.gov/prototype-building-models) in the zip files.

**Comment by rraustad** [comment]
We are just going to have to take this on faith as I cannot easily compare differences. At least it was the authors of the DOE publication that made these changes, that is as good as one would expect. I pulled this branch and even the warehouse is so different (using WinDiff) that it would take a hand review of each file to ensure correct data.

Other than a new weather file and replacement of existing example files and updating of the testfiles\CMakeLists.txt. I do not believe there is anything else to update. @Myoldmopar does the new weather file need to be documented anywhere in some file?

**Comment by Myoldmopar** [comment]
I concur about the contents of the files, it's as reliable an author as we could expect.  CI is happy, with Windows failure being an anomaly.  The weather file is fine, and we aren't packaging it up anyway, it's just in our tests.  I will merge this in.  Thanks @lymereJ and @rraustad.

**Comment by lymereJ** [comment]
Thanks to @jungwyoungs who did most of it!

**Comment by jungwyoungs** [comment]
Thanks all.

**Pull request merged**